src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -120,7 +121,10 @@ int endOfAlignmentInContig() {
      *          Note that the format is NOT the same as that used in {@link #toString()}.
      */
     String toPackedString() {
-        return assemblyId + "-" + contigId + ":" + startInAssembledContig + "-" + endInAssembledContig + ":" + referenceInterval.getContig() + ',' + referenceInterval.getStart() + ',' + (forwardStrand ? '+' : '-') + ',' + TextCigarCodec.encode(cigarAlong5to3DirectionOfContig) + ',' + mapQual + ',' + mismatches;
+        return assemblyId + PACKED_STRING_REP_SEPARATOR + contigId + PACKED_STRING_REP_SEPARATOR +
```

Comment:
Use this instead:

```
String.join(PACKED_STRING_REP_SEPARATOR, assemblyId, contigId, startInAssembledContig, ... mapQual, mismatches);
```
For those elements that are not ```CharSequence```s you might need to prepend ```"" + ``` or call ```String.valueOf(x)```.